### DISCUSSION TITLE
js cannot work on Chrome/69

### DISCUSSION THREAD

**Discussion by zengRRR** [nuxt/nuxt#21529] [Questions]
**Environment**
UA: Mozilla/5.0 (Linux; U; Android 9; zh-CN; MHA-AL00 Build/HUAWEIMHA-AL00) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/69.0.3497.100 UWS/3.22.1.255 Mobile Safari/537.36 AliApp(DingTalk/7.0.30) com.alibaba.android.rimet/29879169 Channel/227200 language/zh-CN abi/64 Hmos/1 UT4Aplus/0.2.25 colorScheme/lig

**Reproduction**
https://www.bangkebang.com/
https://openai.com/

**Describe the bug**
Nuxt3 web, js cannot work. (Chrome/69.0.3497.100)

**link:**
https://openai.com/

![image](https://github.com/nuxt/nuxt/assets/127286038/443eb1e9-c2e6-4ffe-bdb1-05d1f5d367ac)

**Comment by warflash** [comment]
Check out vite's docs on browserc compatibility: https://vitejs.dev/guide/build.html#browser-compatibility

**Reply by zengRRR** [reply]
Ths, so what can I do to solve this problem

**Reply by rafifn** [reply]
you can try this workaround https://github.com/nuxt/nuxt/issues/15464#issuecomment-1469700272
it's works well for me

**Comment by xjccc** [comment]
try
```
vite: {
    build: {
      target: 'es2015'
    }
}
```
